docs/source/documentation.rst

```diff
@@ -328,42 +328,86 @@ Before computing the morphometrics of an image, make sure it has been segmented
+This generates a **'77_axon_morphometrics.xlsx'** file in the image directory::
 
     --image1_sem/
     ---- 77.png
     ---- 77_seg-axon.png
     ---- 77_seg-axonmyelin.png
     ---- 77_seg-myelin.png
+    ---- 77_axon_morphometrics.xlsx
     ---- pixel_size_in_micrometer.txt
-    ---- axon_morphometrics.xlsx
+
     ...
 
 
-Morphometrics of images from multiple folders
+Morphometrics of a specific image from multiple folders
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-To generate morphometrics of images which are located in different folders, specify the path of the image folders using the **-i** argumument of the CLI separated by space. For instance, to compute morphometrics of the images present in the folders **'test_sem_image/image1_sem/'** and **'test_sem_image/image2_sem/'** of the test dataset, use the following command::
+To generate morphometrics of images which are located in different folders, specify the path of the image folders using the **-i** argumument of the CLI separated by space. For instance, to compute morphometrics of the image **'77.png'** and **'image.png'** present in the folders **'test_sem_image/image1_sem/'** and **'test_sem_image/image2_sem/'** respectively of the test dataset, use the following command::
 
     axondeepseg_morphometrics -i test_segmentation/test_sem_image/image1_sem/77.png test_segmentation/test_sem_image/image2_sem/image.png
 
-This will generate **'axon_morphometrics.xlsx'** file in each of folders:: 
+This will generate **'77_axon_morphometrics.xlsx'** and **'image_axon_morphometrics.xlsx'** files in the **'image1_sem'** and **'image2_sem'** folders:: 
+
+    --image1_sem/
+    ---- 77.png
+    ---- 77_seg-axon.png
+    ---- 77_seg-axonmyelin.png
+    ---- 77_seg-myelin.png
+    ---- 77_axon_morphometrics.xlsx
+    ---- pixel_size_in_micrometer.txt
+
+    ...
+
+    --image2_sem/
+    ---- image.png
+    ---- image_seg-axon.png
+    ---- image_seg-axonmyelin.png
+    ---- image_seg-myelin.png
+    ---- image_axon_morphometrics.xlsx
+    ---- pixel_size_in_micrometer.txt
+
+
+Morphometrics of images present in folder(s)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+To compute the morphometrics of batches of images present in folder(s), input the path of the directories using the **-i** argument separated by space. For example, the morphometrics files of the images present in the directories  **'test_sem_image/image1_sem/'** and **'test_sem_image/image2_sem/'** are computed using the following CLI command::
+
+    axondeepseg_morphometrics -i test_segmentation/test_sem_image/image1_sem test_segmentation/test_sem_image/image2_sem
+ 
+This will generate **'77_axon_morphometrics.xlsx'** and **'78_axon_morphometrics.xlsx'** morphometrics files in the **'image1_sem'** directory. And, the **'image_axon_morphometrics.xlsx'** and **'image2_axon_morphometrics.xlsx'** morphometrics files are generated in the **'image2_sem'** directory:: 
```

Comment:
I did not find an image "78.png" in the test_segmentation zip.

Reply:
The images "78.png" and "image2_sem" are not present in the **test_segmentation zip**. I just made up these image names because there were not more than 1 image to test the batch morphometrics feature for users.

To be in sync with the documentation and the **test_segmentation zip**, I guess it would be a better idea to update the segmentation folder with images "78.png" and "image2_sem".

Reply:
I just updated the test_segmentation.zip where "78.png" is a duplicate image of "77.png", and "image2.png" is a duplicate image of "image.png".

Here is the updated test_segmentation folder:
[test_segmentation.zip](https://github.com/neuropoly/axondeepseg/files/6543501/test_segmentation.zip)

This updated zip file needs to be uploaded on **OSF** for the documentation to be in sync.

Reply:
I don't have the rights to upload to OSF, @mathieuboudreau can you do the upload?

Reply:
@maxwab you checked out the file a long time ago, and OSF is not letting me force-check it in (I click the button but nothing happens). I am prevented from uploading a new version while it's checked out. Could you try and see if OSF will let you check it back in unchanged?

<img width="1437" alt="Capture d’écran 2021-06-07 à 11 36 23" src="https://user-images.githubusercontent.com/1421029/121031381-dc443480-c784-11eb-87fc-b08794915aa4.png">